docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2807,6 +2807,284 @@ have not been included. These are:
 #. We sample 100% of events. This is likely unrealistic, but the percent is probably very high still. 
 #. There are errors in SSN data, which are not modeled here. 
 
+Noise Functions
+^^^^^^^^^^^^^^^
+
+In order to have a realistic challenge with PRL, it is essential to have 
+noise added to the data. Noise is generally divided into two types: 
```

Comment:
Yeah, Nathaniel pointed out that there is data type or data formatting noise that we haven't included. Like dates or zip codes can be formatted in different ways. I will adjust the language here in case we want to add it later!

Reply:
> Yeah, Nathaniel pointed out that there is data type or data formatting noise that we haven't included. Like dates or zip codes can be formatted in different ways.

I think this type of noise would be column noise in our terminology. If we haven't figured out how to do it yet, maybe we should add a to-do noting that we will want to include this additional type of column noise later.